src/web/server/htmlTemplate.ts

```diff
@@ -103,6 +103,34 @@ export const htmlTemplate = ({
 
     const twitterMetaTags = generateMetaTags(twitterData, 'name');
 
+    // Duplicated prefetch and preconnect tags from DCP:
```

Comment:
Is it worth adding a link to the DCP code? Or maybe we can just add a link to the mdn docs so this stands by itself?

https://developer.mozilla.org/en-US/docs/Web/Performance/dns-prefetch

Or both! 🤷

Reply:
👍